user_guide/data_management_view.rst

```diff
@@ -0,0 +1,173 @@
+.. _user-guide-ui-components-view-pages:
+
+View Pages
+==========
+
+To see more details of a specific record, and to work directly with the record (create a task related to a customer, 
+appoint a calendar event for a user, turn a cart into an order, share a contact and so on), you need to get to its 
+View page.
```

Comment:
To see more details of a specific record, and to work directly with the record (i.e., create a task related to a customer, appoint a calendar event for a user, turn a cart into an order, share a contact, and so on), you need to get to its View page.